.github/workflows/test-runner-on-perf.yml

```diff
@@ -1,12 +1,91 @@
-# This is a basic workflow to help you get started with Actions
-
 name: Test Runner on Perf Tests
-
+env:
+  GOOGLE_APPLICATION_CREDENTIALS: /tmp/jade-dev-account.json
+  GOOGLE_SA_CERT: /tmp/jade-dev-account.pem
+  GOOGLE_CLOUD_PROJECT: broad-jade-perf
+  GOOGLE_CLOUD_DATA_PROJECT: broad-jade-perf-data2
+  TEST_RUNNER_SERVER_SPECIFICATION_FILE: perf.json
+  GOOGLE_ZONE: us-central1
+  K8_CLUSTER: jade-master-us-central1
 on:
   workflow_dispatch: {}
+  #schedule:
+  #  - cron: '0 2 * * *' # run at 2 AM UTC.
```

Comment:
TODO BEFORE MERGING: switch this to a scheduled run.

Reply:
And if we do end up using perf SAM for the perf env we should check in w QA about when we can run to avoid their testing

Reply:
Good idea!